hooks/use-fetch-exr.ts

```diff
@@ -1,5 +1,5 @@
-import { useQuery } from "@tanstack/react-query";
-import { useMemo } from "react";
+import { ONE_DAY_MS } from "@/lib/date";
```

Comment:
there's already one 🙂 

```suggestion
import { ONE_DAY } from "@/lib/constants";
```